src/build-depcache-output/resolve-required-dependency-scripts/index.js

```diff
@@ -0,0 +1,31 @@
+import ac from 'argument-contracts';
+import { ExternalPackageEntry } from '../../external-package-entry';
+import { Logger } from '../../logger';
+
+function getScripts({ externalPackageEntry, version }) {
+    ac.assertType(externalPackageEntry, ExternalPackageEntry, 'externalPackageEntry');
+    ac.assertString(version, 'version');
+
+    Logger.info(`Creating scripts for ${externalPackageEntry.packageName}`);
+
+    return externalPackageEntry.getEsmUrls(version);
+}
+
+export function resolveRequiredDependencyScripts({ dependencies, dependencyMap }) {
+    ac.assertType(dependencies, Object, 'dependencies');
+    ac.assertType(dependencyMap, Object, 'dependencyMap');
+
+    const scriptsArray = dependencies.reduce((scriptsArray, dependency) => {
```

Comment:
Nitpick: I  like to distinguish the name of my accumulator for readability purposes, so seeing two `scriptsArray` kinda throws me off haha

Reply:
Good call! I agree, that will definitely help readability. I'll make a change